checks.d/mysql.py

```diff
@@ -896,13 +896,19 @@ def _get_stats_from_innodb_status(self, db):
         try:
             with closing(db.cursor()) as cursor:
                 cursor.execute("SHOW /*!50000 ENGINE*/ INNODB STATUS")
-                innodb_status = cursor.fetchone()
-                innodb_status_text = innodb_status[2]
         except (pymysql.err.InternalError, pymysql.err.OperationalError, pymysql.err.NotSupportedError) as e:
             self.warning("Privilege error or engine unavailable accessing the INNODB status \
                          tables (must grant PROCESS): %s" % str(e))
             return {}
 
+        if cursor.rowcount < 1:
```

Comment:
I've just double checked the privileges that the datadog user has, and can confirm `PROCESS` is included (and no different from the other Aurora cluster members):

```
mysql> show grants;
+------------------------------------------------------------------------------------------------------------------------------+
| Grants for datadog@IP                                                                                                        |
+------------------------------------------------------------------------------------------------------------------------------+
| GRANT PROCESS, REPLICATION CLIENT ON *.* TO 'datadog'@'IP' IDENTIFIED BY PASSWORD '*HASH'                                    |
| GRANT SELECT ON `performance_schema`.* TO 'datadog'@'IP'                                                                     |
+------------------------------------------------------------------------------------------------------------------------------+
2 rows in set (0.01 sec)

mysql> show engine innodb status;
Empty set (0.00 sec)
```